modules/core/shared/src/main/scala/gem/Program.scala

```diff
@@ -4,13 +4,13 @@
 package gem
 
 import gem.math.Index
-import scala.collection.immutable.TreeMap
+import scala.collection.immutable.SortedMap
```

Comment:
I have changed a few `TreeMap/Set`s to `SortedMap/Set` to avoid writing codecs for `TreeMap/Set` that we'll just end up throwing away.